.github/workflows/pytest.yml

```diff
@@ -0,0 +1,25 @@
+name: Python test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Can we change this to 18.04?
So this would match with our appliance.

Reply:
I'll check that out

Reply:
fixed in e2fb651